src/ts/tags.ts

```diff
@@ -95,12 +95,12 @@ document.addEventListener('DOMContentLoaded', () => {
     });
   };
 
-  if (document.querySelector('.createTagInputFavorite')) {
-    document.querySelector('.createTagInputFavorite').addEventListener('blur', event => {
+  if (document.getElementById('.createFavTagInput')) {
```

Comment:
why is it a "getElementById" but targets a class?

Reply:
I missed the dot while replacing.